src/service/NotificationService.js

```diff
@@ -80,6 +80,9 @@ const getNotificationBody = (messages) => {
 		return t('mail', '{from}\n{subject}', {
```

Comment:
not related to your change but l10n is absolutely the wrong tool here. we can just concatenate the three strings.